javascript/statements.json

```diff
@@ -433,114 +443,6 @@
             "standard_track": true,
             "deprecated": false
           }
-        },
-        "reassignment_fails": {
-          "__compat": {
-            "description": "Reassignment fails",
-            "support": {
-              "webview_android": {
-                "version_added": true
-              },
-              "chrome": {
-                "version_added": "20"
-              },
-              "chrome_android": {
-                "version_added": true
-              },
-              "edge": {
-                "version_added": true
-              },
-              "edge_mobile": {
-                "version_added": true
-              },
-              "firefox": {
-                "version_added": "13"
-              },
-              "firefox_android": {
-                "version_added": "13"
-              },
-              "ie": {
-                "version_added": "11"
-              },
-              "ie_mobile": {
-                "version_added": "11"
-              },
-              "nodejs": {
-                "version_added": true
-              },
-              "opera": {
-                "version_added": true
-              },
-              "opera_android": {
-                "version_added": true
-              },
-              "safari": {
-                "version_added": true
-              },
-              "safari_ios": {
-                "version_added": true
-              }
-            },
-            "status": {
-              "experimental": false,
-              "standard_track": true,
-              "deprecated": false
-            }
-          }
-        },
-        "allow_in_sloppy_mode": {
```

Comment:
@elchi3 Didn't you removed too much here? Shouldn't "allow_in_sloppy_mode" be kept?

Reply:
See the above comment.
> Also removed the "sloppy mode" entry. I think it is non-sense (I removed it elsewhere already).

Reply:
Oh missed it. Fine with it.